corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -705,8 +705,8 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
                         }
                     }
                 };
-                if (element.hasOwnProperty('binding') && element.hasOwnProperty('ix') && allChildren &&
-                    allChildren.length > 0) {
+                if (Object.prototype.hasOwnProperty.call(element, 'binding') &&
+                    Object.prototype.hasOwnProperty.call(element, 'ix') && allChildren && allChildren.length > 0) {
```

Comment:
nit: underscore methods could make this cleaner
```suggestion
                if (_.has(element, 'binding') && _.has(element, 'ix') && !_.isEmpty(allChildren)) {
```

Reply:
applied - I love underscore now